### PR TITLE
Update to Django 5.0

### PR DESCRIPTION
**Pull request by ricardobranco777**
Update to Django 5.0

### CODE REVIEW

File: ocw/templates/ocw/instance_list.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% load render_table from django_tables2 %}
 {% load static %}
-{% load bootstrap4 %}
+{% load django_bootstrap5 %}
```
Comment (m-dati) [inline ocw/templates/ocw/instance_list.html:4]: houldn't it be `{% load django-bootstrap5 %}` ?

Comment (ricardobranco777) [reply ocw/templates/ocw/instance_list.html:4]: > Shouldn't it be `{% load django-bootstrap5 %}` ?

No. See Shouldn't it be `{% load django-bootstrap5 %}` ?

Comment (m-dati) [reply ocw/templates/ocw/instance_list.html:4]: ok :)

File: ocw/apps.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from django.apps import AppConfig
 from apscheduler.schedulers.background import BackgroundScheduler
 from apscheduler.executors.pool import ThreadPoolExecutor
-from pytz import utc
+from datetime import timezone
```
Comment (mimi1vx) [inline ocw/apps.py:6]: nice , use stdlib :D

**Review by mimi1vx — APPROVED** [review]

**Review by asmorodskyi — APPROVED** [review]
LGTM

**Pull request merged**